### PR TITLE
add description and repos to membership.md

### LINKED ISSUE #585: Add introduction and repos to membership.md for easy program parsing
Like https://github.com/tikv/community/tree/master/teams/pd

### PR DESCRIPTION
**Pull request by zhangyangyu**
close #585

### CODE REVIEW

File: teams/bigdata/membership.json [hunk 1]
```diff
@@ -17,5 +21,12 @@
         "windtalker"
     ],
 
-    "reviewers": []
+    "reviewers": [],
+
+    "repositories": [
+        "tidb-incubator/TiBigData",
+    	"tispark",
```
Comment (Mini256) [inline teams/bigdata/membership.json:28]: Please keep the indent uniform. Same below.

**Review by Mini256 — CHANGES_REQUESTED** [review]

File: teams/bigdata/membership.json [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
         "marsishandsome",
         "sunxiaoguang"
     ],
-
-    "committers": [
-        "Coderlxl",
-        "LittleFall",
-        "ilovesoup",
-        "windtalker"
+    "description": [
```
Comment (tisonkun) [inline teams/bigdata/membership.json:21]: could be a string?

**Review by tisonkun — COMMENTED** [review]
Why do you reorder sections? I'd prefer to append items and prevent mix "refactor"s into changes.

Also for repositories, in the markdown description I suggest default org is "pingcap", but if you'd like to use it in mechanical logic, you may think of always writing "orgname/reponame".

File: teams/bigdata/membership.json (hunk 2 above)
Comment (zhangyangyu) [reply teams/bigdata/membership.json:21]: Just follow the `_comment` style. I don't understand why `_comment` is a list.

Comment (tisonkun) [reply teams/bigdata/membership.json:21]: You're right 😄 . We should correct it later or if you don't mind it, include in this PR.

**Comment by zhangyangyu** [comment]
@tisonkun @Mini256 Mind to give some updates?

**Review by tisonkun — APPROVED** [review]

File: teams/docs/membership.json [hunk 3]
```diff
@@ -54,5 +54,12 @@
         "tiancaiamao",
         "weekface",
         "zimulala"
+    ],
+
+    "repositories": [
+        "docs",
+        "docs-cn",
+        "docs-dm",
+        "docs-tidb-operator"
```
Comment (Mini256) [inline teams/docs/membership.json:63]: Docs Team has created a new Repo: `docs-appdev`.

Refer: https://github.com/pingcap/community/pull/591

**Review by Mini256 — COMMENTED** [review]
rest LGTM.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**